.travis.yml

```diff
@@ -109,7 +109,7 @@ jobs:
     script:
     - curl -X POST -d "docs" https://api.netlify.com/build_hooks/${NETLIFY_HOOK_ID}
   - stage: deploy
-    if: branch = "master" AND tag IS present
+    if: tag IS present
```

Comment:
I guess I'm still confused why the original didn't create a stage since it was on branch master with the tag present
<img width="255" alt="Screenshot 2019-07-10 at 13 12 55" src="https://user-images.githubusercontent.com/22333512/60964798-90787e80-a314-11e9-9eb1-7bf59125e729.png">

Reply:
I think the issue is entirely on the Travis side. The [docs](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables) mention that "Note that for tags, git does not store the branch from which a commit was tagged.", when talking about the environment variable `TRAVIS_BRANCH`. [This person](https://stackoverflow.com/questions/54929914/travis-deploy-stage-not-running-with-condition-if-branch-master-and-tag-is-pr) had the same problem as us (only he had an additional `AND type = push` condition).

Reply:
ahhh i see okay. This makes sense then, I was confused because of [this line](https://github.com/RasaHQ/rasa/pull/3977/files#diff-354f30a63fb0907d4ad57269548329e3R107) but there I suppose the OR takes care of it. 👍

Reply:
Exactly, the branch will be master or a tag will be present